lib/iris/coord_systems.py

```diff
@@ -840,17 +842,33 @@ def __init__(self, longitude_of_projection_origin=0, ellipsoid=None):
                     True longitude of planar origin in degrees.
             * ellipsoid
                     :class:`GeogCS` defining the ellipsoid.
+            * min_latitude
+                    the maximum southerly extent of the projection.
+                    Defaults to -80 degrees.
```

Comment:
Where do these come from? If cartopy, I'd discourage default at all for these kwargs - they aren't fundamental parts of the reference system AFAIKS.

Reply:
Actually, GRIB GDT10 doesn't define min/max latitude, so we could drop these again (we only really need the true scale latitude). Otherwise yes, these come from cartopy.